customer-wallet/src/containers/Payment/PaymentForm.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+// components
+import { CardHeader, CardContent, CardActions } from '@material-ui/core'
+import { Avatar, Button } from '@material-ui/core'
+import { reduxForm, Field } from 'redux-form'
+import Input from 'components/Form/Input'
+import StarRatingComponent from 'react-star-rating-component'
+// icons
+import ReviewIcon from '@material-ui/icons/Star'
+
+function PaymentForm({ handleSubmit, setRating, rating, starCount, disabled }) {
+    return <form onSubmit={handleSubmit}>
+        <CardHeader
+            avatar={<Avatar><ReviewIcon/></Avatar>}
+            title='Leave a Chlu Review'
+            subheader='The review will be included with the payment. You will be able to edit the review later'
+        />
+        <CardContent>
+            <StarRatingComponent
+                className='review-rating'
+                name='rating'
+                starCount={starCount}
+                value={rating}
+                editing={!disabled}
+                onStarClick={rating => setRating(rating)}
+            />
+            <Field
+                name='review'
+                type='text'
+                component={Input}
+                placeholder='Your thoughts...'
```

Comment:
Change to "Review" only.

We will probably change the UI of the Ratings and Review section a bit soon. I think we want bigger stars and dedicated headings to the two sections. We'll get to it later, so maybe we can leave this as it is for now.